From e4ea9eeb969601898041d89786221796791ddaee Mon Sep 17 00:00:00 2001
From: Huidae Cho <grass4u@gmail.com>
Date: Mon, 27 Apr 2026 22:30:21 -0600
Subject: [PATCH] build: Add gui scripts to ALL_MODULES

---
 CMakeLists.txt | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e9d5aadf381..62d49823b47 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -284,16 +284,17 @@ set(ALL_SUBDIRS
 foreach(d ${ALL_SUBDIRS})
   add_subdirectory(${d})
 endforeach()
+
+if(WITH_GUI)
+  add_subdirectory(gui)
+endif()
+
 add_custom_target(
   ALL_MODULES
   DEPENDS ${modules_list}
   COMMENT "Building all modules.")
 set_target_properties(ALL_MODULES PROPERTIES FOLDER Misc)
 
-if(WITH_GUI)
-  add_subdirectory(gui)
-endif()
-
 if(WITH_DOCS)
   add_subdirectory(doc)
   add_subdirectory(man)
