--- include/Make/Install.make.orig	2025-11-21 08:20:09.000000000 -0700
+++ include/Make/Install.make	2026-01-12 22:46:34.099654480 -0700
@@ -101,8 +101,10 @@
 	-rm $(DESTDIR)$(INST_DIR)/$(GRASS_NAME).tmp
 	$(MAKE) $(STARTUP)
 
+ifndef CROSS_COMPILING
 	-rm $(DESTDIR)$(INST_DIR)/$(FONTCAP)
 	$(MAKE) $(DESTDIR)$(INST_DIR)/$(FONTCAP)
+endif
 
 	-rm $(DESTDIR)$(INST_DIR)/$(TMPGISRC)
 	$(MAKE) $(DESTDIR)$(INST_DIR)/$(TMPGISRC)
